pubspec.yaml

```diff
@@ -71,6 +71,9 @@ flutter:
     - assets/images/ui/play_button.png
     - assets/images/ui/try_again.png
     - assets/images/ui/menu_button.png
+    - assets/images/ui/ios_share.png
```

Comment:
Assets used only for CoronaJump (not MonsterJump) should be marked, for example by placing in an extra folder in assets: `assets/cj/`.